README.md

```diff
@@ -6,39 +6,17 @@ ISPyB backend server based on FastAPI.
 
 ## Getting started
 
-You need to have `python >= 3.10`.
+For dependencies, see [https://ispyb.github.io/py-ispyb/](https://ispyb.github.io/py-ispyb/).
 
-Install requirements with
+For installation instructions, see [https://ispyb.github.io/py-ispyb/run.md](https://ispyb.github.io/py-ispyb/run.md)
```

Comment:
This link does not work. I think it should be `/run` instead of `/run.md`.

Reply:
Thank you, good point! Have fixed that now.